assets/sass/_content.scss

```diff
@@ -23,6 +23,8 @@
     h2 {
       border-bottom: 2px solid #f0f0f0;
       padding-bottom: 12px;
+			position: relative;
+			z-index: -1;
```

Comment:
I don't understand this fix (and I couldn't repro the original issue). `z-index: -1` sounds very hacky. Can you move this fix to another PR to discuss first?